client/app/scripts/liveblog-edit/styles/liveblog-edit.less

```diff
@@ -511,12 +511,18 @@
         padding: 10px 0px 10px 0px;
         display: inline-block;
     }
-    .timeline-item-actions {
-        padding-top:25px;
+    .timeline-item-remove {
+        padding-top:9px;
         margin-right:10px;
         float: right;
         display: none;
     }
+    .timeline-post-actions {
```

Comment:
even if is similar we should keep classes for different styles.
maybe just have `display: none;` for `.timeline-item-remove` and define them in common.